l10n_it_withholding_tax_financial_report/pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
```

Comment:
In questa versione non è abilitato, vedi https://github.com/OCA/l10n-italy/blob/a7408f3d84e52d4e69fa8abbd0edc96768c31e49/.copier-answers.yml#L23